video/agon.h

```diff
@@ -212,6 +212,8 @@ enum AudioState : uint8_t {	// Audio channel state
 #define BUFFERED_REVERSE				0x18	// Reverse the order of data in a buffer
 #define BUFFERED_COPY_REF				0x19	// Copy references to blocks from multiple buffers into one buffer
 #define BUFFERED_COPY_AND_CONSOLIDATE	0x1A	// Copy blocks from multiple buffers into one buffer and consolidate them
+#define BUFFERED_COMPRESS				0x1C	// Compress blocks from multiple buffers into one buffer
+#define BUFFERED_DECOMPRESS				0x1D	// Decompress blocks from multiple buffers into one buffer
 
 #define BUFFERED_DEBUG_INFO				0x20	// Get debug info about a buffer

```

Comment:
in hindsight I think one of the mistakes I made with the original buffered commands API was to not make the numbering a bit more logical, grouping things together better, and leaving gaps for future expansion

with that in mind I think it would probably make sense to number these new commands at a different "block" within the buffered command space.  maybe at &40 and &41 ?  BDPP could maybe go in at &30?

the debug info command is undocumented and only there for us devs tinkering around with the buffered commands API, so that could get renumbered to &80 to stay more out of the way

Reply:
That makes sense. Changing the numbers is easy, of course. Just edit and rebuild. You can feel free to do that in the branch, or I will, if you prefer.